### DISCUSSION TITLE
Adding PR#s in towncrier

### DISCUSSION THREAD

**Discussion by shaunporwal** [twisted/towncrier#580] [Q&A] [answered]
Hi! Thanks for this tool, been using it for my python projects and it's much better than recording changes manually! 

Is there some way to automatically add a PR # to the towncrier newsfragments after they form the changelog.md? Perhaps this is something to be done using CI/CD GitHub actions? I know that issue number is included as the integer that is the prefix of the news fragment.

**Comment by adiroiban** [comment]
Hi. Thanks for the report.

I don't understand the use case or what is required here.

Can you please provide a step by step  description of what you want do , what are your manual steps and what are the steps that you expect towncrier to automaticaly do.

Cheers

**Reply by shaunporwal** [reply]
For example when I'm creating the changelog using towncrier create -c 'message' #.type, I want to also have some way to include the pull request number in addition to the issue number. Perhaps this is only something that can be done manually after creation of the pull request?

**Reply by adiroiban** [reply, marked as answer]
>  Perhaps this is only something that can be done manually after creation of the pull request?

If the PR is not created yet... then we don't know the PR ID

Maybe some convention like `create -c 'message' 1234-5678.type` where 1234 is the Issue ID and 5678 is the PR ID ... but you will have to add custom code for supporting this use case

Most of the time, I create the towncrier info before creating the PR.

And, if in the PR description you add `Fixes #1234` you will see a link to the PR from the Issue page.

**Reply by shaunporwal** [reply]
Thank you! I think the last one is a good no-code solution
